samples/client-provider/query.fsx

```diff
@@ -0,0 +1,37 @@
+﻿#r "../../src/FSharp.Data.GraphQL.Client/bin/Debug/FSharp.Data.GraphQL.Client.dll"
+
+open FSharp.Data.GraphQL
+open System.Collections.Generic
+
+let [<Literal>] serverUrl = "http://localhost:8083"
+
+// The name and arguments of the query will be automatically set by the type provider
+let [<Literal>] queryFields = "{ id, name, appearsIn, friends { name } }"
+let [<Literal>] queryFieldsWithFragments = "{ ...data, friends { name } } fragment data on Human { id, name, appearsIn }"
+
+type MyClient = GraphQLProvider<serverUrl>
+
+let hero =
+    MyClient.QueryHero<queryFieldsWithFragments>("1000")
```

Comment:
In this case you're creating statically typed top level query field. However I think it would be good to still leave the possibility of specifying it from string, like `{ hero("1000") { id, name, appearsIn, friends { name } } }`.

Reason for that is that user may define his own top level query and use multiple queries defined in schema as part of single execution. Take a look at [MutationTests](https://github.com/bazingatechnologies/FSharp.Data.GraphQL/blob/dev/tests/FSharp.Data.GraphQL.Tests/MutationTests.fs#L77) for examples.